cpp/src/arrow/util/compression_lz4.cc

```diff
@@ -41,6 +41,9 @@ namespace util {
 
 namespace {
 
+constexpr int kLZ4MinCompressionLevel = 1;
+constexpr int kLZ4DefaultCompressionLevel = 1;
```

Comment:
The same constant is already defined in the `.h`, why redefine it here?

Reply:
Yes that's a duplicate, I'll remove it. Actually, wouldn't it be nicer to move the min/max values into the `.h` file too? (also see `compression_bz2.h`, it has the min/max values in the `.cc` file)